notebooks/multimodal-rag/multimodal-rag-llamaindex.ipynb

```diff
@@ -0,0 +1,1168 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    demo = make_demo(filepath, build_index, search, generate, load_image, stop)
</pre><p>There are no <code>load_image </code>argument in <code>make_demo </code>function.﻿</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>